packages/babel-plugin-jsx-dom-expressions/README.md

```diff
@@ -1,273 +1,296 @@
-# Babel Plugin JSX DOM Expressions

```

Comment:
I'm not really sure what happened here... the only changes is additional lines starting at 178

Reply:
Ah that makes sense. Happy to revert the newline changes if that's preferred.